packages/ia-components/.storybook/middleware.js

```diff
@@ -0,0 +1,37 @@
+const proxy = require('http-proxy-middleware');
```

Comment:
I'd refactor this file to make it easier to both change the base URL for the API (in the event of a dev or staging env) and to quickly add routes, like:

```
const base_url = "https://archive.org";

const constructOptions = (route) => (
  {
    target: `${base_url}/${route}`,
    logLevel: 'debug',
    changeOrigin: true,
    pathRewrite: {
      `^/${route}`: `${base_url}/${route}`
    }
  }
);

module.exports = function expressMiddleware (router) {
  ["download", "images", "serve"].forEach((route) => {
    router.use(`/${route}`, proxy(constructOptions(route)))
  });

  router.use('/api', proxy({
    target: base_url,
    logLevel: 'debug',
    changeOrigin: true,
    pathRewrite: {
      '^/api': base_url
    }
  }));
};
```

Reply:
Great contribution, Shane. Agree.

Reply:
that's a good optimization, will apply it.